Library/Formula/radare2.rb

```diff
@@ -38,24 +32,56 @@ class Radare2 < Formula
   depends_on "valabind" => :build
   depends_on "swig" => :build
   depends_on "gobject-introspection" => :build
+  depends_on "gmp"
   depends_on "libewf"
   depends_on "libmagic"
-  depends_on "gmp"
   depends_on "lua51" # It seems to latch onto Lua51 rather than Lua. Enquire this upstream.
   depends_on "openssl"
+  depends_on "yara"
 
   def install
     # Build Radare2 before bindings, otherwise compile = nope.
     system "./configure", "--prefix=#{prefix}", "--with-openssl"
-    system "make"
+    system "make", "CS_PATCHES=0"
```

Comment:
If `CS_PATCHES=` it will try to download a tarball of patches for capstone during build-time.  If I remember correctly it also attempts to run `git reset --hard` which I think caused the build to fail unless you were installing from `HEAD`.